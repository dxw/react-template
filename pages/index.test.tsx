import renderer from "react-test-renderer";

import Index from "./index";

it("renders correctly", () => {
  const component = renderer.create(<Index />);

  expect(component.toJSON()).toMatchSnapshot();
});
