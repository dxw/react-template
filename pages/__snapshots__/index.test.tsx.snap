// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  Welcome to dxw's React template!
</h1>
`;
